xbmc/dbwrappers/mysqldataset.cpp

```diff
@@ -145,7 +146,13 @@ int MysqlDatabase::connect(bool create_new) {
   if (host.empty() || db.empty())
     return DB_CONNECTION_NONE;
 
-  //CLog::Log(LOGDEBUG, "Connecting to mysql:%s:%s", host.c_str(), db.c_str());
+  std::string resolvedHost;
+  if (CDNSNameCache::Lookup(host, resolvedHost))
+  {
+    CLog::Log(LOGDEBUG, "{} replacing configured host {} with resolved host {}", host,
+              resolvedHost);
```

Comment:
three times `{}` is too much for two logvars. this is exactly where it stuck :)
fix is fine if this is corrected:
`DEBUG <general>: replacing configured host dummy with resolved host 192.168.200.105`

Reply:
:) thanks, fixed